### DISCUSSION TITLE
How to integrate reviews for product

### DISCUSSION THREAD

**Discussion by kAgotsi** [saleor/saleor#10219] [Q&A] [answered]
Hello,

Thanks for the sailor community for this great open source e-commerce product.

Today in product, are there existant reviews product module ?
if not how to developed it ( can we pass through app integrations features)?

Thanks

**Comment by karolkielecki** [comment, marked as answer]
Hi @kAgotsi, thanks for asking and your kind words!

Saleor doesn't support it. Indeed, you can use apps for that. Learn more about extending here: https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts. The question is how advanced/specific such an app would be. Every use case is different. 

You can also integrate just your fronted with a product reviews tool without creating an app and, e.g., use metadata to store additional info about products. Learn more about metadata here: https://docs.saleor.io/docs/3.x/developer/metadata

**Comment by kAgotsi** [comment]
Thanks you

**Reply by jwhawkes** [reply]
What did you end up doing @kAgotsi ? Did you find a solution? I'm in the situation where my client needs this. I look forward to your reply.

**Reply by timuric** [reply]
@jwhawkes this is a popular choice among many merchants https://trustpilot.com/

You could also create a simple review system by storing reviews in product metadata
